### PR TITLE
Mention support for button entity type in Android Power Menu

### PR DESCRIPTION
**Pull request by jpelgrom**
Documentation PR for home-assistant/android#2134

### CODE REVIEW

File: docs/integrations/android-power-menu.md [hunk 1]
```diff
@@ -9,10 +9,12 @@ The ![Android](/assets/android.svg) Android app will automatically integrate wit
 Currently support is limited to the following domains:
 
 *  `automation` On/Off
+*  `button` Press
```
Comment (dshokouhi) [inline docs/integrations/android-power-menu.md:12]: Let's add a beta label for these additions since they wont be available until we cut a release

**Review by dshokouhi — CHANGES_REQUESTED** [review]

File: docs/integrations/android-power-menu.md (hunk 1 above)
Comment (jpelgrom) [reply docs/integrations/android-power-menu.md]: Like this?

Comment (dshokouhi) [reply docs/integrations/android-power-menu.md]: you could drop the logo if you want since this page is only for Android but yes :)

**Review by dshokouhi — APPROVED** [review]
Looks good to me, can be merged when parent is merged

**Pull request merged**